jme3-core/src/main/java/com/jme3/input/FlyByCamera.java

```diff
@@ -382,28 +382,7 @@ protected void rotateCamera(float value, Vector3f axis){
      * @param value zoom amount
      */
     protected void zoomCamera(float value){
-        // derive fovY value
-        float h = cam.getFrustumTop();
-        float w = cam.getFrustumRight();
-        float aspect = w / h;
-
-        float near = cam.getFrustumNear();
-
-        float fovY = FastMath.atan(h / near)
-                / (FastMath.DEG_TO_RAD * .5f);
-        float newFovY = fovY + value * 0.1f * zoomSpeed;
-        if (newFovY > 0f) {
-            // Don't let the FOV go zero or negative.
-            fovY = newFovY;
-        }
-
-        h = FastMath.tan( fovY * FastMath.DEG_TO_RAD * .5f) * near;
-        w = h * aspect;
-
-        cam.setFrustumTop(h);
-        cam.setFrustumBottom(-h);
-        cam.setFrustumLeft(-w);
-        cam.setFrustumRight(w);
+        cam.setFov(cam.getFov() - value * 0.1F * zoomSpeed);

```

Comment:
Shouldn't that be `cam.getFov() + value * 0.1F * zoomSpeed`?

Reply:
You would think so. However, when I did that, scrolling up would zoom out. That didn't strike me as very intuitive, so I flipped it.

Reply:
Changing the semantics of `FlyByCamera.setZoomSpeed()` would be a different PR.

Reply:
Right, I just double-checked the behavior with the master branch, and scrolling up does in fact zoom out. I thought it was something I changed by mistake, but apparently its "intended" behavior. I'll use the plus sign.